src/parse-tsconfig/resolve-extends.ts

```diff
@@ -25,7 +25,7 @@ function resolveFromPackageJsonPath(packageJsonPath: string) {
 	);
 }
 
-export function resolveExtends(
+export function resolveExtendsPath(
```

Comment:
Revert

Reply:
I changed it because its name conflicted with the new function created. I don't have any idea of new naming of the function. Can you help it?